docs/gallery/advanced_io/plot_zarr_io.py

```diff
@@ -10,11 +10,10 @@
 
 Note that the Zarr native storage formats are optimized for storage in cloud storage
 (e.g., S3). For very large files, Zarr will create many files which can lead to
-issues for traditional file systems (that are not cloud object stores) due to
-limitations on the number of files per directory (this affects local disk,
-GDrive, Dropbox etc.).
+issues for traditional file system (that are not cloud object stores) due to limitations
+on the number of files per directory (this affects local disk, GDrive, Dropbox etc.).
 
-Zarr read and write is provided by the :hdmf-zarr:`hdmf-zarr package<>`. First, create an
+Zarr read and write is provided by the :hdmf-zarr:`hdmf-zarr` package. First, create an
```

Comment:
```suggestion
Zarr read and write is provided by the :hdmf-zarr:`hdmf-zarr<>` package. First, create an
```

This change results in a broken link in https://pynwb--1835.org.readthedocs.build/en/1835/tutorials/advanced_io/plot_zarr_io.html#sphx-glr-tutorials-advanced-io-plot-zarr-io-py
but it works on the dev branch build
https://pynwb.readthedocs.io/en/dev/tutorials/advanced_io/zarr_io.html#sphx-glr-tutorials-advanced-io-zarr-io-py